### DISCUSSION TITLE
[question]: how to convert lng & lat to x, y ,z

### DISCUSSION THREAD

**Discussion by casperiv0** [TGRHavoc/live_map#64] [Q&A]
Hey!

This is somewhat related to #51. I've tried to convert the lat & lng from the `moveend` event in leaflet using `map.project({ lat, lng })` but with no success. I think there is a simple solution but I can't come up with it, any suggestions how I could convert it? 

Example:
```js
marker.on("moveend", (e) => {
 const latLng = e.target._latlng;
 
 // This would return the converted values from the leaflet map back to the GTA map
 const xYZ = ConvertLatLngToXYZ(latLng);
})

```

Thanks!

**Comment by TGRHavoc** [comment]
Sorry for the late reply!

Anyways, you can find the (not so good) code to translate [in-game coords to leaflet's lat-lang in the interface source code](https://github.com/TGRHavoc/live_map-interface/blob/master/js/src/utils.1.js#L56). 

You might be able to just reverse the operations used here to convert back to game-coords.

Problem: they might not be accurate.

**Comment by casperiv0** [comment]
Yeah, it doesn't seem to be very accurate. Unless I did some wrong calculation lol
